### PR TITLE
Committed files for initial review.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
This is the code for a fluid component that provides a couchdb-like REST interface to pouchdb for testing purposes.

It requires the gpii-express module also currently under review.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Global Public Inclusive Infrastructure
```
Comment (amb26) [inline LICENSE:1]: See gpii-express review for copyright comment

File: src/js/pouch.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Utility functions to add pouch to an existing express instance
+"use strict";
+var fluid          = require('infusion');
+var namespace      = "gpii.pouch";
+var pouch          = fluid.registerNamespace(namespace);
+var PouchDB        = require("pouchdb");
+var memdown        = require('memdown');
+var expressPouchdb = require('express-pouchdb');
+
+pouch.addRoutesPrivate = function(that) {
```
Comment (amb26) [inline src/js/pouch.js:10]: As before in gpii-express - break up and rename the functionality in this initialisation block

File: src/tests/all-tests.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            jqUnit.start();
+            isSaneResponse(jqUnit, error,response, body);
+
+            var data = (typeof body === "string") ? JSON.parse(body) : body;
+            jqUnit.assertTrue("There should be records.", data.doc_count && data.doc_count > 0);
+        });
+    });
+
+    // TODO:  test inserts
+    jqUnit.asyncTest("Testing insertion of a new record...", function() {
```
Comment (amb26) [inline src/tests/all-tests.js:93]: As before with gpii-express, express this repetitive test material declaratively

File: src/tests/log.txt [hunk 4]
```diff
@@ -0,0 +1,67 @@
+[Tue, 03 Feb 2015 13:30:02 GMT] [info] [<0.000.0>] 127.0.0.1 - - GET / 200
```
Comment (amb26) [inline src/tests/log.txt:1]: This log file appears to have been committed in error

**Comment by the-t-in-rtf** [comment]
Abandoning this pull request in favor of one that's actually against the right repo.  Will still respond here in context as I make requested changes.

File: src/tests/log.txt (hunk 4 above)
Comment (the-t-in-rtf) [inline src/tests/log.txt:1]: Done.

File: LICENSE (hunk 1 above)
Comment (the-t-in-rtf) [inline LICENSE:1]: Done.

**Pull request closed without merging**